orcid/en/appendix-testing-orcid-sandbox.md

```diff
@@ -0,0 +1,50 @@
+3. Some of the information in the registration form (e.g., name) will be automatically populated based on the information set to public visibility in the user’s ORCID Record.
+4. Once registered and logged in, the user can go to their Profile and click on the Public tab to see their authenticated ORCID iD. **Please Note:** There is currently [a bug associated with this feature](https://github.com/pkp/orcidProfile/issues/158). Once you submit a publication and the publication gets published, the ORCID iD should appear on the publication in OJS/OPS. If you are using the Member API, the publication should also appear on the ORCID Record in the works section once your article is published (please note: this does not currently work with OPS).
+
+#### *Scenario 2*: An article with one author is submitted by the author while logged in to their OJS/OPS author account. 
+1. If an author has an OJS/OPS user account where they are identified as an author, and can sign in to OJS/OPS, their ORCID iD can be connected to OJS/OPS via their OJS/OPS user profile:
+2.  When logged in to OJS/OPS, go to your Profile and click on the Public tab
+3. Click the button to “Create or Connect your ORCID iD” - you will be prompted to sign in to your ORCID record (make sure to use your sandbox ORCID email address and password) and “authorize” OJS/OPS to access your ORCID iD and ORCID record.
+4. Your authenticated ORCID iD should then appear.
+5. Once you submit a publication and the publication gets published, the ORCID iD should appear on the publication in OJS/OPS. If you are using the Member API, the publication should also appear on the ORCID record in the works section once your article is published (please note: this does not currently work with OPS).
+
+#### *Scenario 3:* An article with multiple authors is submitted by one author while logged in to their OJS/OPS author account. 
+1. The author who is submitting the article can connect their own ORCID iD in their OJS/OPS profile by following the instructions listed in Scenario 1.
+2. To request authenticated ORCID iDs for the other authors/contributors, follow the steps in Scenario 4.
+
+#### *Scenario 4*: An article with one or more authors is submitted by an administrator or other individual who is not one of the authors.
+
+1. During the article submission process, when you get to step 3 “Enter Metadata,” there is a section to add contributors. 
+2. For each contributor, click “Add Contributor” and enter the information for each author. For the purposes of testing in the sandbox, you will want to make sure to use the `@mailinator.com` email address that you used to create your sandbox ORCID iD.
+3. Towards the bottom of the “Add Contributor” form, in the “ORCID” section, you will need to check the box to Send e-mail to request ORCID authorization from contributor. Then click Save.
+
+![Image of ORCID contributor authorization](./assets/orcid-contributor-authorization.png)
```

Comment:
Since this image is in a list, the line breaks before and after should be removed so it will render properly (otherwise it will start a new list after the line breaks).

Reply:
Noted. Have modified in https://github.com/pkp/pkp-docs/pull/596/commits/71a7780526466eb30513d0dd4a5189ab44b64701